components/NavigationLink/NavigationLink.js

```diff
@@ -0,0 +1,29 @@
+import styled, { css } from "styled-components";
+
+export const NavigationLinkWrapper = styled.span`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 100%;
+  width: 50px;
```

Comment:
Just a suggestion, but you could replace px with a more responsive unit like em and rem. If you want to make sure you keep this exact styling you can use [this tool](https://nekocalc.com/de/px-zu-rem-umrechner) to convert everything. :)

Reply:
Will do , tnx!